examples/bri-3/src/bri/zeroKnowledgeProof/models/proof.ts

```diff
@@ -6,9 +6,7 @@
  */
 
 export class Proof {
-  a: string[];
-  b: string[][];
-  c: string[];
+  value: object;
```

Comment:
can we add type for `value` here?

Reply:
Different zk protocols have different values/parameters that are generated inside a proof. I have kept it as a generic object to accommodate different protocols. And while verifying the proof, we need to maintain the exact format in which it was generated.